README.md

```diff
@@ -1,6 +1,80 @@
-Authors Haven - A Social platform for the creative at heart.
-[![CircleCI](https://circleci.com/gh/andela/metis-ah-frontend/tree/develop.svg?style=svg)](https://circleci.com/gh/andela/metis-ah-frontend/tree/develop) [![Test Coverage](https://api.codeclimate.com/v1/badges/93530611b5b0d23beebf/test_coverage)](https://codeclimate.com/github/andela/metis-ah-frontend/test_coverage) [![Build Status](https://travis-ci.org/andela/metis-ah-frontend.svg?branch=develop)](https://travis-ci.org/andela/metis-ah-frontend)
-=======
+## Authors Haven - A Social platform for the creative at heart.
+
+
 ## Vision
 Create a community of like minded authors to foster inspiration and innovation
 by leveraging the modern web.
+
+## Metis-ah-frontend 
+[![CircleCI](https://circleci.com/gh/andela/metis-ah-frontend/tree/develop.svg?style=svg)](https://circleci.com/gh/andela/metis-ah-frontend/tree/develop) [![Test Coverage](https://api.codeclimate.com/v1/badges/93530611b5b0d23beebf/test_coverage)](https://codeclimate.com/github/andela/metis-ah-frontend/test_coverage) [![Build Status](https://travis-ci.org/andela/metis-ah-frontend.svg?branch=develop)](https://travis-ci.org/andela/metis-ah-frontend)
+
+
+## Description
+
+This project creates a platform for Authors to create and share articles. It is equally a platform that allows user to read articles, react to them and socialize with one another.
+
+### Website url
+https://metis-ah-frontend.herokuapp.com/
```

Comment:
Well done for providing the link to viewing the application, you could also use a markdown like this ```[View live app Here](https://metis-ah-frontend.herokuapp.com/)``` [View live app Here](https://metis-ah-frontend.herokuapp.com/)